.circleci/config.yml

```
@@ -471,40 +487,30 @@ jobs:
       CC: clang
       CXX: clang++
       MAKEFLAGS: -j 10
-    steps:
-      - checkout
-      - run: *run_build
-      - store_artifacts: *artifacts_solc
-      - persist_to_workspace: *artifacts_executables
+    <<: *steps_b_ubu
 
-  b_ubu_asan_clang: &b_ubu_asan_clang
-    docker:
-      - image: << pipeline.parameters.ubuntu-2004-clang-docker-image >>
-    environment:
-      CC: clang
-      CXX: clang++
-      CMAKE_OPTIONS: -DSANITIZE=address
-      MAKEFLAGS: -j 3
-    steps:
-      - checkout
-      - run: *run_build
-      - store_artifacts: *artifacts_solc
-      - persist_to_workspace: *artifacts_executables
-
-  b_ubu_ubsan_clang: &b_ubu_ubsan_clang
+  b_ubu_au_clang:
```

Let's use `san` here. The job is meant to be used for sanitizers.
```suggestion
  b_ubu_san_clang:
```

---

Done.